src/Powercord/plugins/pc-hastebin/index.js

```diff
@@ -12,7 +12,7 @@ module.exports = class Hastebin extends Plugin {
     this.registerCommand(
       'hastebin',
       [],
-      'Lets you paste content to Hastebin.',
+      'Lets you paste content to Hastebin',
       '{c} [ --send ] < --clipboard | FILE_URL >',
```

Comment:
Might as well close up the spacing between these arguments for additional sake of consistency.
```suggestion
      '{c} [--send] <--clipboard | FILE_URL>',
```